qcelemental/physical_constants/ureg.py

```diff
@@ -28,33 +28,53 @@ def build_units_registry(context):
     ureg.define(
         "hartree_inverse_meter = {} / hartree / m".format(phys_const["hartree-inverse meter relationship"]["value"])
     )
+    ureg.define("plancks_constant = {} * joule * s".format(phys_const["planck constant"]["value"]))
```

Comment:
Did we want `h/hbar` here as well?

Reply:
I added `h` and `hbar` to constants in #200 and asked why we have a separate unit registry from the constants. Can we continue this discussion there?

Reply:
Can the unit be singular like the constant? planck, not plancks

EDIT: ah, I see it was like that before, drat.

Reply:
Maybe add an alias? It should be singular.

Reply:
Alias should be above `plank_constant = planks_constant = {} * joule...`